finetune/util/in_memory_finetune.py

```diff
@@ -79,12 +79,12 @@ def after_run(self, run_context, run_values):
     def end(self, session):
         self._evaluate(session)
 
-def make_in_memory_finetune_hooks(finetune):
+def make_in_memory_finetune_hooks(finetune, estimator):
```

Comment:
Calling this finetune like the pkg name makes me a little bit uncomfortable.  Maybe just model?